adwords_client/internal_api/builder.py

```diff
@@ -96,10 +96,20 @@ def _parse_operation(self, operation, sync=None):
                 yield {}
             elif object_type == 'batch_job':
                 yield from self._parse_batch_job_operation(operation)
+            elif object_type == 'offline_conversion' and sync:
+                yield from self._parse_offline_conversion_operation(operation)
             else:
                 logger.warning('Operation not recognized: {}', operation)
                 yield None
 
+    def _parse_offline_conversion_operation(self, operation):
+        if 'fields' in operation or 'default_fields' in operation:
+            raise NotImplementedError()
```

Comment:
_could_  this be implemented in the future, or having 'fields' or 'default_fields' will always be an error?

Asking because `NotImplementedError` means "I do not know how to do it now, but is possible."
However, `ValueError` means "You had called it with wrong values. You are asking nonsense things", for exemple.